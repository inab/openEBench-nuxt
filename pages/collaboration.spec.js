describe('Collaboration', () => {
	test.todo('should be implemented');
});
