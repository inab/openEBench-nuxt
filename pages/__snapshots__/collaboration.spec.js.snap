// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collaboration should match snapshot 1`] = `
"<v-container-stub tag=\\"div\\">
  <h1 class=\\"text--h3\\">OpenEBench Collaboration</h1> <br>
  <p>
    Collaborations are at the heart of OpenEBench. OpenEBench has been
    designed as a space to foster collaborations within and across
    communities, taking ELIXIR as the overall framework. ELIXIR is the
    pan-European Research Infrastructure for managing research data and other
    digital assets in the Life Sciences domain.
  </p>
  <p>
    Collaborations are essential to ensure that OpenEBench is fit for its
    purpose and responds to users' current needs. It is also a mechanism to
    contribute towards the platform's sustainability. Participation in EU and
    nationally funded projects is one of the existing collaboration
    mechanisms. Participating in funded projects allows us to prioritise
    specific developments to strengthen, extend and add new features to the
    platform that a specific project will utilise. Notably, these developments
    are made available to everyone across OpenEBench, fostering a continuous
    innovation cycle.
  </p>
  <p>
    Beyond funded projects, OpenEBench also works with scientific communities
    willing to sustain their benchmarking efforts in the long run, as well as
    individuals interested in contributing to specific platform components.
  </p> <br>
  <v-row-stub tag=\\"div\\">
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://bioinfo4women.bsc.es/research/\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">BAIHA</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://www.datatools4heart.eu/\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub height=\\"45px\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"https://www.datatools4heart.eu/wp-content/themes/datatools/css/img/logo_color.svg\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">DataTools4Heart</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://www.deciderproject.eu/\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">DECIDER</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://elixir-europe.org/about-us/how-funded/eu-projects/steers\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"65%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"https://elixir-europe.org/sites/default/files/styles/left-large/public/images/elixir-steers-logo.png\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">ELIXIR STEERS</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://everse.software/\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"https://everse.software/images/logos/EOSCEverse_PosColour.svg\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">EOSC-EVERSE</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://cancerimage.eu/\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">EUCAIM</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://eucanimage.eu/\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">EuCanImage</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://dev-openebench.bsc.es/benchmarking/OEBC900?event=OEBE9000000000\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">Fujitsu AMetaPred</h3>
      </v-card-stub>
    </v-col-stub>
    <v-col-stub cols=\\"12\\" sm=\\"6\\" md=\\"4\\" tag=\\"div\\">
      <v-card-stub loaderheight=\\"4\\" href=\\"https://permedcoe.eu/\\" tag=\\"div\\" target=\\"_blank\\" maxheight=\\"350\\" maxwidth=\\"344\\" minheight=\\"250\\" hover=\\"true\\" class=\\"mx-auto mb-5\\">
        <div class=\\"image-container\\">
          <v-img-stub maxwidth=\\"80%\\" options=\\"[object Object]\\" position=\\"center center\\" src=\\"https://permedcoe.eu/wp-content/uploads/2020/11/logo_1.png\\" transition=\\"fade-transition\\" class=\\"responsive-img\\"></v-img-stub>
        </div>
        <v-divider-stub></v-divider-stub>
        <h3 class=\\"text-center mt-3\\">PerMedCoE</h3>
      </v-card-stub>
    </v-col-stub>
  </v-row-stub>
</v-container-stub>"
`;
